x-pack/docs/en/security/get-started-security.asciidoc

```diff
@@ -168,15 +168,16 @@ Select a role to see more information about its privileges. For example, select
 the `kibana_system` role to see its list of cluster and index privileges. To
 learn more, see <<privileges-list-indices>>. 
 
-Let's assign the `kibana_user` role to your user. Go back to the 
-*Management / Security / Users* page and select your user. Add the `kibana_user` 
+Let's assign the `kibana_admin` role to your user. Go back to the 
+*Management / Security / Users* page and select your user. Add the `kibana_admin` 
```

Comment:
Note: This getting started doc intentionally uses `kibana_admin`, and then directs users to explore the range of Kibana Authz options.